client/extensions/woocommerce/components/custom-tree-selector/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import CustomTreeNode from './custom-tree-node';
+
+function renderPlaceholders() {
+	return (
+		<div>
+			<div className="custom-tree-selector__placeholder" />
+			<div className="custom-tree-selector__placeholder" />
+			<div className="custom-tree-selector__placeholder" />
+		</div>
+	);
+}
+
+const CustomTreeSelector = ( {
```

Comment:
Do you think we need the "custom" prefix? I think `TreeSelector` would be fine on its own. (for the entire component & file names too)

Reply:
Yeah, I suppose you're right. If this is destined for calypso proper especially, this makes sense.